data_base.sql

```diff
@@ -0,0 +1,250 @@
+-- SQLINES DEMO *** iaDB 10.1.48-MariaDB-0+deb9u2 dump
+
+/* SET NAMES utf8; */
+-- time_zone := '+00:00';
+/* SET foreign_key_checks = 0; */
+/* SET sql_mode = 'NO_AUTO_VALUE_ON_ZERO'; */
+
+DROP TABLE IF EXISTS Background;
+-- SQLINES FOR EVALUATION USE ONLY (14 DAYS)
+CREATE TABLE Background (
+  Background_id int NOT NULL,
+  Image varchar(255) NOT NULL,
+  Gradient varchar(100) NOT NULL,
+  Background varchar(255) NOT NULL,
+  Room varchar(255) NOT NULL
+) ;
+
+
+DROP TABLE IF EXISTS Effect;
+CREATE TABLE Effect (
+  Effect_id int NOT NULL,
+  Name varchar(100) NOT NULL,
+  Type varchar(50) NOT NULL,
+  HP int DEFAULT NULL,
+  Power int DEFAULT NULL,
+  Stamina int NOT NULL,
+  PRIMARY KEY (Effect_id)
+) ;
```

Comment:
Remove this extra space here and everywhere
```suggestion
);
```